code/game/gamemodes/clock_cult/clock_ratvar.dm

```diff
@@ -192,8 +206,10 @@
 	world << "<span class='heavy_brass'><font size=5>\"[pick("BLOOD GOD!!!", "NAR-SIE!!!", "AT LAST, YOUR TIME HAS COME!")]\"</font></span>"
 	world << "<span class='cult'><font size=5>\"<b>Ratvar?! How?!</b>\"</font></span>"
 	for(var/obj/singularity/narsie/N in range(15, src))
-		clash_of_the_titans(N) //IT'S TIME FOR THE BATTLE OF THE AGES
+		if(N.clashing)
+			continue
 		N.clashing = TRUE
+		clash_of_the_titans(N) //IT'S TIME FOR THE BATTLE OF THE AGES
```

Comment:
This proc sleeps. Accordingly, it didn't set Nar-Sie to clashing until after it finished.